chroma_core/migrations/0015_device.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2020-04-15 13:20
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chroma_core", "0014_multipletimesyncalert_notimesyncalert_timeoutofsyncalert"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Device",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
```

Comment:
Do we need an id field? I think fqdn is the primary key

Reply:
Yeah the fqdn should always be unique. That makes sense to me.

Reply:
Character fields complicate integration with `warpdrive`, we had that issue with `Device` having character `id` previously.

Reply:
I'm ok with it then.